platform/android/build.gradle

```diff
@@ -13,11 +13,14 @@ allprojects {
     repositories {
         jcenter()
         maven { url "http://oss.sonatype.org/content/repositories/snapshots/" }
+        flatDir {
+            dirs '../MapboxGLAndroidSDK/libs'
+        }
     }
 }
 
 task wrapper(type: Wrapper) {
-    gradleVersion = '3.2.1'
+    gradleVersion = '2.14.1'
```

Comment:
Gradle 3 contains significant bug fixes that we require to build Mapbox GL as part of the Gradle build process. We can't downgrade to Gradle 2.